specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/examples/AvailableDelegationsSubscriptionGet.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters" : {
+    "api-version": "2018-08-01",
+    "location": "regionName",
+    "subscriptionId" : "subId"
+  },
+  "responses" : {
+    "200" : {
+      "body" : {
+        "value": [
+          {
+            "name": "Microsoft.Provider.resourceType",

```

Comment:
"id", "name", "type" are not properties of "AvailableDelegation"

Reply:
fixed.